sunpy/map/tests/test_plotting.py

```diff
@@ -345,6 +345,17 @@ def test_plot_autoalign(aia171_test_map):
     return fig
 
 
+@figure_test
+def test_plot_autoalign_reproject(aia171_test_map):
+    aia171_test_map._data = aia171_test_map.data.astype('float32')
+    rotated_map = aia171_test_map.rotate(30*u.deg, order=3)
+
+    fig = Figure()
+    ax = fig.add_subplot(projection=aia171_test_map)
+    rotated_map.plot(axes=ax, autoalign='reproject')
+    return fig
```

Comment:
I think my issue with this figure test is that it's hard to know if it worked or we just overlaid the second map on top.

Can we also change the color map and maybe pass in an alpha?

Reply:
> I think my issue with this figure test is that it's hard to know if it worked or we just overlaid the second map on top.
> 
I followed this [Auto_Aligning](https://docs.sunpy.org/en/stable/generated/gallery/map_transformations/autoalign_aia_hmi.html) and tested this code where I used **autoalign='reproject'** and it gave same result as the one in gallery example 
```
fig = plt.figure(figsize=(12, 5))
ax1 = fig.add_subplot(121, projection=map_aia)
map_aia.plot(axes=ax1, clip_interval=(1, 99.9)*u.percent)
ax2 = fig.add_subplot(122, projection=map_aia)
map_hmi.plot(axes=ax2, autoalign='reproject', title='HMI image in AIA reference frame')
ax2.axis(ax1.axis())
```
![image](https://github.com/user-attachments/assets/3457b4dd-6f39-4ed6-aaac-813d23d702f0)

> Can we also change the color map and maybe pass in an alpha?

In the test right? Okay working on it